core/src/main/scala/tech/beshu/ror/adminapi/AdminRestApi.scala

```diff
@@ -152,6 +153,7 @@ object AdminRestApi extends Logging {
   val updateIndexConfigurationPath: Path = Path.create(NonEmptyList.of("_readonlyrest", "admin", "config" ))
   val provideRorFileConfigPath: Path = Path.create(NonEmptyList.of("_readonlyrest", "admin", "config", "file"))
   val provideRorIndexConfigPath: Path = Path.create(NonEmptyList.of("_readonlyrest", "admin", "config"))
+  val provideRorConfigPath: Path = Path.create(NonEmptyList.of("_readonlyrest", "admin", "config", "load"))
```

Comment:
I see new method here. I don't see we agreed new API with Kibana plugin side. How it will be used?

IMO Kibana's plugin neeeds two methods:
* get config
  * return current configuration
  * return information is all nodes have configured the same type of config and the same config 
* save index config

And there are more - we should agree how:
* user is going to bootstrap new cluster from scratch (if he starts the cluster with file ROR config, will he be able to edit it in Kibana plugin?)
* it seems that we should block modification when:
 * we find inconsistent configuration - a) one node has config C1, other one C2; b) one node uses index config, other file config
 * file config is only used

@sscarduzio @here-s-johnny what do you think?

Reply:
We didn't agree. I wanted to know which states are possible, and how communication looks like. To me it was clear after tests. 
Each node declares it's own configuration, or error. If we get two different configurations, we can't say which is correct. Ex two nodes could return config from two different indices. In this case we're unable to say which config is correct, so modification should be blocked, and explanation should be presented to user. I'll prepare some table with possibilities. 

Do we allow user to have different file configs on nodes?